### ISSUE TITLE
Missing Models

### ISSUE THREAD

**Issue by sleepybot5** [catthatcodes/car-damage-assessment#6]
![missing_models](https://github.com/shubhi/car-damage-assessment/assets/41358015/a1489c8d-0ccc-45f7-bf27-5614f2655363)
 How do we access Line number 20-22 models? They aren't attached in the repo

**Comment by pedroaugustoramos** [comment]
Yes, I have the same question. Or should we train the models ourselves. If you could share your pre-trained models it would be greatly appreciated. Thank you!

**Comment by Muhammadyousafrana** [comment]
it's useless because there isn't any model in the repo❌

**Comment by Munira923** [comment]
have you successfully run the code?

**Comment by Muhammadyousafrana** [comment]
> have you successfully run the code?
> 

Yes

**Comment by pedroaugustoramos** [comment]
Yes! Although I need more training data to get better model results. Thank
you!

Pedro Ramos
(55)11-7558-7855


Em sáb., 21 de dez. de 2024 às 17:11, MUHAMMAD YOUSAF RANA <
***@***.***> escreveu:
